crates/sui-adapter/src/object_root_ancestor_map.rs

```diff
@@ -35,14 +34,13 @@ impl ObjectRootAncestorMap {
                 stack.push(cur_id);
                 match cur_owner {
                     Owner::ObjectOwner(parent_id) => {
-                        cur_owner = *direct_owner_map.get(&parent_id.into()).ok_or(
-                            SuiError::MissingObjectOwner {
-                                child_id: cur_id,
-                                parent_id: parent_id.into(),
-                            },
-                        )?;
+                        cur_owner = *direct_owner_map
+                            .get(&parent_id.into())
+                            .ok_or(ExecutionErrorKind::MissingObjectOwner)?;
```

Comment:
Might be helpful to specify which one for the missing object owner, but not blocking